### PR TITLE
Added lime-mojito drink

### PR DESCRIPTION
**Pull request by iMoose**
Just added an extra drink to the repo

### CODE REVIEW

File: src/recipes/lime-mojito.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "Fill the glass almost to the top with ice",
+        "Pour the rum over the ice and fill the glass with soda water",
+        "Garnish with the remaining lime wedge",
+    ],
+    "image": "lime-mojito.jpg",
+    "source": "http://allrecipes.com.au/recipe/5330/real-lime-mojito.aspx",
+    "keywords": [
+        "alcoholic",
+        "lime",
+        "sugar",
```
Comment (alfg) [inline src/recipes/lime-mojito.json]:
```suggestion
        "sugar"
```

File: src/recipes/lime-mojito.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "ingredient": "Soda water"
+        },
+    ],
+    "directions": [
+        "Place mint leaves and 1 lime wedge into a glass",
+        "Use a muddler to crush the mint and lime to release the mint oils and lime juice",
+        "Add 2 more lime wedges and the sugar, and muddle again to release the lime juice. Do not strain the mixture.",
+        "Fill the glass almost to the top with ice",
+        "Pour the rum over the ice and fill the glass with soda water",
+        "Garnish with the remaining lime wedge",
```
Comment (alfg) [inline src/recipes/lime-mojito.json]:
```suggestion
        "Garnish with the remaining lime wedge"
```

File: src/recipes/lime-mojito.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        {
+            "quantity": "1",
+            "measure": "cup",
+            "ingredient": "Ice cubes"
+        },
+        {
+            "quantity": "125",
+            "measure": "milliliters(ml)",
+            "ingredient": "Soda water"
+        },
```
Comment (alfg) [inline src/recipes/lime-mojito.json]:
```suggestion
        }
```

**Comment by iMoose** [comment]
> Possible to provide a higher quality image at 600x400px?

Higher quality image uploaded

**Comment by iMoose** [comment]
> 600x400px without stretching the image, please. If it's closer to square, that's ok.

Updated to a different image without streching

**Comment by alfg** [comment]
Perfect! Thanks!

**Comment by iMoose** [comment]
Thanks, sorry for the hassle

**Comment by alfg** [comment]
Not a hassle at all! Thanks for contributing!

**Pull request merged**